Cargo.toml

```diff
@@ -379,9 +379,9 @@ ntapi = { git = "https://github.com/MSxDOS/ntapi.git", rev = "24fc1e47677fc9f6e3
 
 [features]
 # Default features for *-unknown-linux-gnu and *-apple-darwin
-default = ["api", "api-client", "enrichment-tables", "sinks", "sources", "sources-dnstap", "transforms", "unix", "rdkafka?/gssapi-vendored", "enterprise", "component-validation-runner"]
+default = ["api", "api-client", "enrichment-tables", "sinks", "sources", "sources-dnstap", "transforms", "unix", "rdkafka?/gssapi-vendored", "enterprise"]
```

Comment:
Also, it's not clear to me why we list `sources-dnstap` explicitly here? What is special about this source? It seems like it should be listed in `sources-logs` and removed from here.

Reply:
`sources-dnstap` is only compiled on *nix targets.